src/org/dux/cli/DuxBuildTracer.java

```diff
@@ -168,6 +163,23 @@ private void parseStraceFile(boolean includeProjDir, DuxTraceBlacklist blacklist
             String path = rawPath.substring(1, rawPath.length() - 1);
             DuxCLI.logger.debug("got path: {}", path);
 
+            // if this file is in the C:\Windows directory, don't need to store
+            String os = System.getProperty("os.name");
+            if (os.startsWith("Windows")) {
+                String[] parts = path.split(":"); // ["C", "\Windows\..."]
+                if (parts.length == 1) {
```

Comment:
From experimentation, it looks like string split returns an array with at least on element. E.g. "test" gets split on ":" as ["test"]. Even the empty string gets split as [""]. I can't find a way to get a 0-element array back.

My reason for doing == 1 was because it reminded me of a lecture Hal Perkins gave once where he was iterating through some data structure with a for loop. Rather than putting "< size" as the test, he put "!= size". Someone asked him why he did that, and he said it's better to be precise about your assumptions, and if something is wrong, to fail early, rather than be too general and not realize serious erros. I.e. in the example of looping through a data structure, if you loop from 0 up to the size of a data structure in steps of 1 and terminate at an index much greater than the size (rather than exactly equal to the size), then something has gone seriously wrong and it is better to catch that case rather than just terminate as if all is well.

I think it's an interesting argument he makes, but I'm not really sure what I would do if I got back a 0-length array from string split here. That would mean something is wrong with the library... So I have changed this to the way you recommended.